### PR TITLE
Update samples with fixes

### PR DESCRIPTION
**Pull request by richlander**
Same fixes as https://github.com/Microsoft/dotnet-framework-docker-samples/pull/24

### CODE REVIEW

**Comment by MichaelSimons** [comment]
Why not update the clone steps of the alpine-preview sample?  Also I think the changes to use the `out` directory during publish should be uniformly applied.

**Comment by richlander** [comment]
Responded to your feedback.

I realized that my base for this branch was super old, so I had to rebase it with my master. Not sure how that happened. Had to force push as a result. I was missing the alpine directory in my branch.

**Review by MichaelSimons — APPROVED** [review]
LGTM - Nit, I think the slashes should be changed in the selfcontained .dockerignore to be consistent with the changes you are introducing here.

File: dotnetapp-prod-alpine-preview/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
...
 
-This .NET Core Docker sample demonstrates a best practice pattern for building Alpine based Docker images for .NET Core apps for production. The primary goal of Alpine is very small deployments.  Images can be pulled quicker and will have a smaller attack surface area.  The .NET Core Alpine Docker images are currently in preview. See the [.NET Core Alpine Docker Image announcement](https://github.com/dotnet/dotnet-docker-nightly/issues/500) for additional details.
+This .NET Core Docker sample demonstrates a best practice pattern for building Alpine based Docker images for .NET Core apps for production.
+
+The primary goal of Alpine is very small deployments.  Images can be pulled quicker and will have a smaller attack surface area.  The .NET Core Alpine Docker images are currently in preview. See the [.NET Core Alpine Docker Image announcement](https://github.com/dotnet/dotnet-docker-nightly/issues/500) for additional details.
 
 The [sample Dockerfile](Dockerfile) creates an .NET Core application Docker image based off of the [.NET Core Runtime Alpine Preview Docker image](https://hub.docker.com/r/microsoft/dotnet-nightly/).
```
Comment (ravimeda) [inline dotnetapp-prod-alpine-preview/README.md]: Wording `an .NET` Core should be `a .NET` Core

**Review by ravimeda — APPROVED** [review]

**Pull request merged**